fineract-provider/src/main/java/org/apache/fineract/infrastructure/accountnumberformat/data/AccountNumberFormatDataValidator.java

```diff
@@ -44,9 +47,12 @@
 import org.springframework.stereotype.Component;
 
 @Component
+@Data
+@NoArgsConstructor
+@Accessors(chain = true)
```

Comment:
Why the Accessors annotation?
The NoArgsConstructor can be switched over to RequiredArgsConstructor if the field is kept as final.

Reply:
done